packages/fast-components-class-name-contracts/src/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from "./button";
+export * from "./toggle";

```

Comment:
Likely should alphabetize

Reply:
I think the components should be together, and then the "other" stuff should be separate (managed-classes in this case)

Reply:
In theory, Foundation is a component.

Reply:
Synced with @nicholasrice  on this. In the index.ts files where we import all things in "src" we will alphabetize. This does not need to be the case in .tsx files for regular imports. Unfortunately, due to some weird configs with .ts lint, we'll need to enforce this by principle. Trying to enforce alphabetizing in these files using .tslint would create a massive hinderance to our dev experience and overall code readability

Reply:
Got it.